inc/ThirdParty/Themes/Jevelin.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace WP_Rocket\ThirdParty\Themes;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+class Jevelin implements Subscriber_Interface {
+	/**
+	 * Return an array of events that this subscriber wants to listen to.
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		return [
+			'rocket_rucss_inline_content_exclusions' => 'preserve_patterns',
+		];
+	}
+
+	/**
+	 * Preserves the CSS patterns when adding the used CSS to the page
+	 *
+	 * @since 3.11.3
+	 *
+	 * @param array $patterns Array of patterns to preserve.
+	 *
+	 * @return array
+	 */
+	public function preserve_patterns( $patterns ): array {
+		if ( ! self::is_jevelin() ) {
+			return $patterns;
+		}
+
+		$preserve = [
+			'#heading-',
+		];
+
+		return array_merge( $patterns, $preserve );
```

Comment:
```suggestion
		$patterns[] = '#heading-';

		return $patterns;
```

Better to avoid using array_merge when it's not absolutely necessary

Reply:
@Tabrisrp True, I gave this a thought but then I also thought about a scenario where we might be preserving more than just one. what do you think?

Reply:
We can always update the code later if needed 👍🏼

Reply:
ok Awesome!